wdl-ast/src/v1/expr.rs

```diff
@@ -66,6 +66,28 @@ pub enum Expr {
 }
 
 impl Expr {
+    /// Attempts to reference a literal expression.
```

Comment:
All of these were not strictly needed for this PR, but I did need one, so I added all of them to be consistent.